lib/Dialect/FIRRTL/InstanceGraph.cpp

```diff
@@ -89,3 +89,48 @@ InstanceGraphNode *InstanceGraph::getOrAddNode(StringRef name) {
 Operation *InstanceGraph::getReferencedModule(InstanceOp op) {
   return lookup(op.moduleName())->getModule();
 }
+
+ArrayRef<InstancePath> InstancePaths::getAbsolutePaths(Operation *op) {
+  assert((isa<FModuleOp, FExtModuleOp>(op))); // extra parens makes parser smile
```

Comment:
isa<FModuleLike>, the new hotness.

Reply:
You could just take a FModuleLikeOp and make it the caller's problem.

Reply:
I'll leave this as is to prevent changes to the original `InstanceGraphCache` code.